lib/preview/queue.ex

```diff
@@ -56,7 +75,16 @@ defmodule Preview.Queue do
 
     case key_components(key) do
       {:ok, package, version} ->
-        create_package(package, version)
+        files = create_package(package, version)
+
+        all_versions = all_versions(package)
+        version = Version.parse!(version)
+
+        if Preview.Utils.latest_version?(version, all_versions) do
```

Comment:
We can get the latest version from the same hexpm API call we do to get all versions.

Reply:
Although from a second look I see we need the `latest_version?` for `paths_for_sitemaps` so do whatever is cleanest.

Reply:
Although from a second look I see we need the latest_version? for paths_for_sitemaps so do whatever is cleanest.